lib/formdata.c

```diff
@@ -789,6 +789,12 @@ static CURLcode setname(curl_mimepart *part, const char *name, size_t len)
   return res;
 }
 
+/* wrap call to fseeko so it matches the calling convetion of callback */
+static int fseeko_wrapper(void *stream, curl_off_t offset, int whence)
+{
+  return fseeko(stream, (off_t)offset, whence);
```

Comment:
Isn't `fseeko()` going to cause portability problems? For example I don't think it exists on Windows...

Reply:
fseeko is [POSIX.1-2008](https://pubs.opengroup.org/onlinepubs/9699919799/functions/fseek.html), but I don't know if that helps on windows.

Lets see what the CI says...